starlite/config/response_cache.py

```diff
@@ -31,31 +29,20 @@ def default_cache_key_builder(request: Request[Any, Any, Any]) -> str:
 
 
 @dataclass
-class CacheConfig:
+class ResponseCacheConfig:
     """Configuration for response caching.
 
     To enable response caching, pass an instance of this class to :class:`Starlite <.app.Starlite>` using the
-    ``cache_config`` key.
+    ``response_cache_config`` key.
     """
 
-    backend: Storage | None = field(default=None)
-    """A :class:`Storage <.storage.base.Storage>`. Defaults to :class:`MemoryStorage <.storage.memory.MemoryStorage>`
-    """
-    expiration: int = field(default=60)
+    default_expiration: int = field(default=60)
     """Default cache expiration in seconds."""
-    cache_key_builder: CacheKeyBuilder = field(default=default_cache_key_builder)
-    """:class:`CacheKeyBuilder <.types.CacheKeyBuilder>`. Defaults to
-    :func:`default_cache_key_builder <.cache.default_cache_key_builder>`.
-    """
-
-    def to_cache(self) -> Cache:
-        """Create a cache wrapper from the config.
-
-        Returns:
-            An instance of :class:`Cache <.cache.Cache>`
-        """
-        return Cache(
-            backend=self.backend or MemoryStorage(),
-            default_expiration=self.expiration,
-            cache_key_builder=self.cache_key_builder,
-        )
+    key_builder: CacheKeyBuilder = field(default=default_cache_key_builder)
+    """:class:`CacheKeyBuilder <.types.CacheKeyBuilder>`. Defaults to :func:`default_cache_key_builder`."""
+    store: str = "request_cache"
+    """Name of the :class:`Store <.stores.base.Store>` to use"""
+
+    def get_store_from_app(self, app: Starlite) -> Store:
+        """Get the store defined in :attr:`store` from an :class:`Starlite <.app.Starlite>` instance"""
```

Comment:
```suggestion
        """Get the store defined in :attr:`store` from an :class:`Starlite <.app.Starlite>` instance."""
```

Reply:
Didn't we have a linting rule in place for this?